ast-psi/src/test/kotlin/kastree/ast/WriterTest.kt

```diff
@@ -29,6 +29,23 @@ class WriterTest {
     }
 
     fun assertParseAndWriteExact(code: String) {
-        assertEquals(code.trim(), Writer.write(Parser.parseFile(code)).trim())
+
```

Comment:
Added the check here, so tests didn't have to be duplicated.  Had mutable visitor tests as a separate file, but made it too easy for tests to diverge.